Modules/Tags.Application/Tags/Queries/ListTags/ListTagsResponse.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Backbone.Modules.Tags.Application.Tags.Queries.ListTags;
+
+[JsonConverter(typeof(ListTagsResponseConverter))]
+public class ListTagsResponse : Dictionary<string, IEnumerable<TagInfo>>
+{
+    public IEnumerable<string> SupportedLanguages =>
+        Values
+            .SelectMany(tags => tags)
+            .SelectMany(tag => tag.DisplayNames.Keys)
+            .Distinct();
+
+    public ListTagsResponse(IReadOnlyDictionary<string, IEnumerable<TagInfo>> tags) : base(tags)
+    {
+    }
+}
+
+public class ListTagsResponseConverter : JsonConverter<ListTagsResponse>
```

Comment:
If I don't miss anything, then a JsonConverter shouldn't be necessary. The JsonSerializer should be able to handle everything on its own - given the model is built correctly.

Reply:
Since the tag and attribute names are properties, they would be converted to camel case when serializing. Therefore I had to write a custom converter.

Reply:
I'm would imagine there is a way to avoid that.